hw-10/main.js

```diff
@@ -0,0 +1,117 @@
+const printMultiplication = number => {
+  if (number < 1 || number > 10) return;
+  console.log('in bang nhan')
+  for (let i = 0; i <= 10; i++) {
+    console.log(`${number} x ${i} = ${number * i}`)
+  }
+}
+//b2 in cac số chẵn
+const printEvenNumber = number => {
+  if (number < 1 || number > 30) return;
+    for (i = 1; i <= number; i++) {
+      if (i % 2 === 0)
+        console.log(i)
+    }
+}
+// b3 tinh tong tu 1-> n
+const getSum = number => {
+  if (number < 1 || number > 30) return;
+  let sum = 0
+  for (let i = 0; i <= number; i++) {
+    sum = sum + i
+  }
+  return sum;
+}
+//b4 tinh giai thua (factorial) 
```

Comment:
```suggestion
}

// b4 tinh giai thua (factorial) 
```
comment tương tự bên trên.
Các block cách nhau ra cho dễ nhìn